Docs for #1736 fluent binder

aldas opened the pull request:
Docs for:

* https://github.com/labstack/echo/pull/1736
* https://github.com/labstack/echo/pull/1734
* https://github.com/labstack/echo/pull/1727

Maybe someone with better English takes time to edit wording etc if needed.

pafuent commented on website/content/guide/request.md:
````diff
@@ -6,12 +6,129 @@ description = "Handling HTTP request in Echo"
+```
+
+### Methods
+`FailFast` flags binder to stop binding after first bind error during binder call chain. Enabled by default.
````
I think FailFast should include the () to be consistent

pafuent commented on website/content/guide/request.md:
```diff
@@ -6,12 +6,129 @@ description = "Handling HTTP request in Echo"
+
+* `<Type>("param", &destination)` - if parameter value exists then binds it to given destination of that type i.e `Int64(...)`.
+* `Must<Type>("param", &destination)` - parameter value is required to exist, binds it to given destination of that type i.e `MustInt64(...)`.
+* `<Type>s("param", &destination)` - (for slices) if parameter values exists then binds it to given destination of that type i.e `I`nt64s(...)`.
```
Here there is a extra \` after the I (\`I\`nt64s(...)\`)

pafuent commented on website/content/guide/request.md:
```diff
@@ -6,12 +6,129 @@ description = "Handling HTTP request in Echo"
+* `<Type>s("param", &destination)` - (for slices) if parameter values exists then binds it to given destination of that type i.e `I`nt64s(...)`.
+* `Must<Type>s("param", &destination)` - (for slices) parameter value is required to exist, binds it to given destination of that type i.e `MustInt64s(...)`.
+
+for some slice types `BindWithDelimiter("param", &dest, ",")` supports slitting parameter values before type conversion is done
```
Typo on slitting

pafuent reviewed (commented):
This PR includes the docs of #169, so maybe this could be merged instead of it if the Fluent Binder PR gets merged soon

lammel commented:
@aldas @pafuent Maybe you can take a look on the doc updates.
Binding has been moved to it's own guide now and feedback from @pafuent is included.

aldas commented:
lgtm. I added 2 little changes:
* example was anonymous function (without name) so I added `e.POST("/users", ` in front of example to be more familiar route definition. 
* mentioned `UnixTimeNano()` as supported fluent binder function

splitting binder to its own page is good move. It is easier to find it on site.
